src/calculate-fps.js

```diff
@@ -0,0 +1,35 @@
+export default (frames, visibilityEvents) =>
+  1000 / (
+    (
+      (frames[frames.length - 1] - frames[0]) -
+      getHiddenDelta(visibilityEvents)
+    ) /
+    (frames.length - 1)
+  )
+
+export const getHiddenDelta = (visibilityEvents) =>
```

Comment:
I have to say, this entire function is an enigma to me. From the method name, I think it takes a number of visibility change events and calculates how much time from the first event to the last that were spent with the tab being hidden, but I'm not certain, and the code is not giving me any hints.

I think it might be helpful if some of these expressions were labelled. See [my other comment](https://github.com/pirelenito/collect-fps/pull/2/files#r95990236) for one example of something that I have no idea _why_ it's being done.